ls1intum/Artemis
# `Development`: Reorganize setup documentation

### Checklist
#### General



### Motivation and Context
Closes #6449.
Closes #5657.

The setup documentation at the moment interleaves documentation that is only relevant for local development setups and documentation that is only relevant for production setups. This makes it hard to follow the documentation in both cases.

### Description
This PR aims to reorganise the sections of the setup documentation so that ones only relevant for production setups (e.g., customisation, privacy statement) are moved to the admin docs.
Additionally, the setup for extension services is also moved to the admin docs. To make sure users are aware of those special services, a note is added to the start of the development docs that explains that this page only explains the core setup but the additional services to extend Artemis functionality are available.

Thus, the actual content of the individual sections has been mostly unchanged (except for small formatting adjustments like adapting the section nesting depth) and only been extracted into separate files and moved into a different directory.

As a future PR the Gitlab/Jenkins setup docs can be further improved to move the nginx-setup instructions from there to the admin section as well. However, in this first step this has not been done, since it would require restructuring within the section, rather than moving entire independent sections from dev to admin docs.

### Steps for Testing
Look at the setup docs for developers and admins and check that the sections you would expect are still there and that images are shown.

- https://artemis-platform--7323.org.readthedocs.build/en/7323/dev/setup.html
- https://artemis-platform--7323.org.readthedocs.build/en/7323/admin/setup.html
- https://artemis-platform--7323.org.readthedocs.build/en/7323/admin/extension-services.html


### Review Progress
#### Code Review
#### Manual Tests
